README.md

```diff
@@ -1 +1,84 @@
-# ecommerce-app-cogito-fe
\ No newline at end of file
+bash```
+# ecommerce-app-cogito-fe
+
+## INTRODUCTION
+We invite you to explore the Cogito E-Commerce frontend repository. Within this repository, you will find the complete codebase for our dynamic and intuitive E-commerce web application. Crafted with ReactJS, TailwindCSS, and Redux, our application delivers an engaging online marketplace where users can seamlessly browse and purchase products. Furthermore, it empowers sellers with efficient tools to effectively manage their product listings.
+
+## MAIN FEATURES
+
+Our E-commerce application is equipped with a comprehensive set of features aimed at enhancing the shopping experience for our users. Here's an overview of the key functionalities:
+
+- login functionality: Allow users to login and provide token according to user's role
+
+- Effortless Shopping Cart: Seamlessly add products to your cart and proceed through a secure checkout process.
+
+- Personalized Wishlist: Create and manage a wishlist of desired products.
+
+- Streamlined Order Management: Efficiently manage and track your orders, ensuring a smooth purchasing journey.
+
+- Convenient Payment Gateway: Safely and conveniently process payments for your orders.
+
+- User Profiles: Maintain your personal information and access your complete order history.
+
+- Intuitive Search and Filtering: Easily find specific products using advanced search and filtering options.
+
+- Extensive Product Catalog: Discover a wide range of high-quality products across various categories.
+
+- Informative Reviews and Ratings: Provide and access insightful feedback and ratings for products.
+
+- notifications: Notify seller when the products is expired and when user's password needs to be updated
+-  public chat: enables real-time communication between users, facilitating product inquiries, customer support, issue resolution, and building trust with you. 
+
+## USING THE PROJECT
+
+### Technologies Used
+
+- Vite React 
+- Tailwind CSS  
+- Redux toolKit
+
+### Dependencies
+
+Please ensure that your Node version is v18 and above. Make sure you have the following dependencies installed: "react" (v18.2.0 or higher), "react-dom" (v18.2.0 or higher), and "tailwindCSS" (version or higher).
+
+### Initial Setup
+Follow these steps to set up the cogito's Frontend codebase locally:
+
+Clone the repository:
+`git clone https://github.com/atlp-rwanda/ecommerce-app-cogito-fe.git`
+
+Go to the folder: 
+`cd ecommerce-app-cogito-fe`
+
+Install yarn Grobally in your local machine
+`npm install -g yarn`
```

Comment:
Do we need to run both commands at the same time?

Reply:
I choose to keep the first one as most known